src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -27,10 +27,13 @@ export default class OrganizationIntegrations extends AsyncComponent {
   getEndpoints() {
     let {orgId} = this.props.params;
 
+    const plugins = ['vsts', 'github', 'bitbucket', 'github_enterprise'];
+    const query = `plugins=${plugins.join('+')}`;
+
     return [
       ['config', `/organizations/${orgId}/config/integrations/`],
       ['integrations', `/organizations/${orgId}/integrations/`],
-      ['plugins', `/organizations/${orgId}/plugins/`],
+      ['plugins', `/organizations/${orgId}/plugins/?${query}`],
```

Comment:
This is a poor comment since I’m on my phone, but you can specify the query as a 3rd item in this array with { query: ... }. Check async component

Reply:
oh nice. 👍